legend-engine-server/src/test/java/org/finos/legend/engine/server/test/shared/PureTestHelper.java

```diff
@@ -119,15 +124,20 @@ protected void tearDown() throws Exception
         };
     }
 
-    public static ServersState initEnvironment() throws Exception
+    public static ServersState initEnvironment(boolean withH2, String serverConfigFilePath) throws Exception
     {
         int engineServerPort = 1100 + (int) (Math.random() * 30000);
         int metadataServerPort = 1100 + (int) (Math.random() * 30000);
         int relationalDBPort = 1100 + (int) (Math.random() * 30000);
 
-        // Relational
-        org.h2.tools.Server h2Server = AlloyH2Server.startServer(relationalDBPort);
-        System.out.println("H2 database started on port:" + relationalDBPort);
+        org.h2.tools.Server h2Server = null;
+
+        if (withH2)
```

Comment:
When is initEnvironment invoked with withH2 = true ?

Reply:
it is the default case for legacy tests on local h2 server